src/main/java/seedu/duke/notebooks/Page.java

```diff
@@ -3,4 +3,14 @@
 public class Page {
     private String title;
     private String content;
+
+    public Page(String title, String content) {
+        this.title = title;
+        this.content = content;
+    }
+
+    @Override
+    public String toString() {
+        return title + ", " + content;
```

Comment:
Potentially put the title and the content on separate lines?

Reply:
Good suggestion!